src/js/components/Button/StyledButtonKind.js

```diff
@@ -246,9 +247,13 @@ const StyledButtonKind = styled.button`
     !props.plain &&
     props.align &&
     `
-  text-align: ${props.align};
-  `}
-  ${props => props.hoverIndicator && hoverIndicatorStyle(props)}
+    text-align: ${props.align};
+    `}
+    ${props => props.hoverIndicator && hoverIndicatorStyle(props)}
+  // plain buttons should still be able to receive active styling even
+  // though they do not have a kindStyle.
+  // other button types are receiving active styling via their kindStyle.
+  ${props => !props.disabled && props.active && props.plain && activeStyle}
```

Comment:
Is the regular button that isn't using `button.default` applying active styling on `plain`?

Reply:
no its not using plain 
I think we can change to 
` ${props => !props.disabled && props.active && activeStyle}`

this still worked for me without the plain

Reply:
Yes, the regular button does apply active styling on `plain`: https://github.com/grommet/grommet/blob/master/src/js/components/Button/StyledButton.js#L203
```
${props => !props.disabled && props.active && activeButtonStyle(props)}
```